src/courtroom.cpp

```diff
@@ -1419,10 +1400,45 @@ void Courtroom::list_areas()
           treeItem->setBackground(1, gaming_brush);
       }
     }
-    else {
-      treeItem->setBackground(1, free_brush);
+    // Brush logic untied from ARUP system, can override.
+    if (i_area.startsWith("^B=")) {
```

Comment:
what does this mean i have never seen this before in my life

Reply:
It means that if the server sends us an area name that starts with ^B=, they're trying to hack in a color in the ^B=1Lobby format.
In hindsight this should probably just made a fuckin packet + update area metadata instead.

Reply:
also, why this syntax? why not something like `\c{1}`